index.html

```diff
@@ -0,0 +1,191 @@
+                    <li class="css">css</li>
+                    <li class="javascript">javascript</li>
+                </ul>
+                <button class="see-project" type="button">See Project</button>
+            </div>
+        </section>
+        <section class="about-me">
+            <h1>About<br>
+                Myself
+            </h1>
+            <p>
+                Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, 
+                don't hestiate to contact me.
+            </p>
+            <h6>LET'S CONNECT</h6>
+                <ul class="social-icons">
+                    <li>
+                        <a href="https://www.twitter.com/son_ofzion"><img src="./assets/images/Vector-3.png" alt="twitter"></a>
+                    </li>
+                    <li>
+                        <a href="https://www.linkedin.com/in/sonofion"><img src="./assets/images/Linkedin-icon.png" alt="linkedin"></a>
+                    </li>
+                    <li>
+                        <a href="https://www.medium.com/@victoradesamuel16"><img src="./assets/images/Vector-1.png" alt="medium"></a>
+                    </li>
+                    <li>
+                        <a href="https://github.com/victoradesamuel"><img src="./assets/images/Vector-2.png" alt="github"></a>
+                    </li>
+                    <li>
+                        <a href="https://hashnode.com/@SonofZion"><img src="./assets/images/Vector.png" alt="hashnode"></a>
+                    </li>
+                </ul>
+                <button class="get-resume" type="button">Get my resume</button>
+                <div class="skills">
+					<div class="languages">
+                        <div class="menu">
+                            <p>Languages</p>
+                                <div class="angle-icon">
+                                <i class="fa fa-angle-left"></i>
+                                </div>
+                        </div>  
+                        <ul class="core">
+                            <li>JavaScript</li>
+                            <li>HTML</li>
+                            <li>CSS</li>
+                        </ul>
+                    </div>
+                    <div class="frameworks">
+						<div class="menu">
+                            <p>Frameworks</p>
+						    <div class="angle-icon">
+                            <i class="fa fa-angle-down"></i>
+						    </div>
+                        </div>
+                    </div>
+                    <div class="non-tech-skills">
+						<div class="menu">
+                            <p>Skills</p>
+						    <div class="angle-icon">
+							<i class="fa fa-angle-down"></i>
+						    </div>
+                        </div>
+                    </div>
+                </div>    
```

Comment:
Please change the `Languages`, `Frameworks` and `Skills` to be `li` tags instead of `div` tags. And the `div` that takes class `languages` to be `ul` tag for semantic purposes.